src/datadog/azext_datadog/manual/commands.py

```diff
@@ -0,0 +1,13 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands import CliCommandType
+
+
+def load_command_table(self, _):
+
+    datadog_link = CliCommandType(operations_tmpl='azure.cli.extension.datadog.link#{}')
+    with self.command_group('datadog link', datadog_link, is_experimental=True) as g:
```

Comment:
```suggestion
    with self.command_group('datadog link', is_experimental=True) as g:
```
The `operations_tmpl` doesn't look right. And you don't need to add a `CliCommandType` here as you only have `custom_command` and it will use the `datadog_custom` defined in `__init__.py`.

https://github.com/Azure/azure-cli-extensions/blob/676abc83e6205e6232e8c223ea515b349a3bfbbc/src/datadog/azext_datadog/__init__.py#L24-L28